### PR TITLE
Update how-can-you-use-zap-to-scan-apis.md

### PR DESCRIPTION
**Pull request by zakrush**
Information was add  about removing of SOAP add-on

**- Summary**

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: site/content/faq/how-can-you-use-zap-to-scan-apis.md [hunk 1]
```diff
@@ -6,14 +6,15 @@ weight: 7
 ---
 
 
-ZAP understands API formats like JSON and XML and so can be used to scan APIs.
```
Comment (psiinon) [inline site/content/faq/how-can-you-use-zap-to-scan-apis.md:9]: ZAP still understands XML, is just doesnt handle SOAP natively.

**Comment by psiinon** [comment]
Thanks - thats a useful update to the docs :)

File: site/content/faq/how-can-you-use-zap-to-scan-apis.md [hunk 2]
```diff
@@ -6,14 +6,15 @@ weight: 7
...
+ZAP understands API formats like JSON and so can be used to scan APIs.
+Earlier ZAP understanded XML format like WSDL. The SOAP Scanner add-on has been removed for the time being.
 
 The problem is usually how to effectively explore the APIs.
 
 There are various options:
 
   * If your API has an OpenAPI/Swagger definition then you can import it using the [OpenAPI add-on](/docs/desktop/addons/openapi-support/).
-  * If your API has a WSDL then you can import it using the SOAP Scanner add-on.
```
Comment (thc202) [inline site/content/faq/how-can-you-use-zap-to-scan-apis.md:16]: Once can still use the SOAP add-on, it's "just" not available in the marketplace. Maybe we should add a link here with the remarks about the issues it has.

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: site/content/faq/how-can-you-use-zap-to-scan-apis.md (hunk 2 above)
Comment (zakrush) [reply site/content/faq/how-can-you-use-zap-to-scan-apis.md:16]: > One can still use the SOAP add-on, it's "just" not available in the marketplace. Maybe we should add a link here with the remarks about the issues it has.
It has some problem to setup add-on that not available in marketplace for DockerZAP and autotests. This requires volume directory with add-os and stores this in remote git and pulling for every host where working ZAP.

Comment (thc202) [reply site/content/faq/how-can-you-use-zap-to-scan-apis.md:16]: Agreed, I'm not saying that's straightforward to get it (compared to marketplace) just that you can still use it (which the removed statement states).

Comment (psiinon) [reply site/content/faq/how-can-you-use-zap-to-scan-apis.md:16]: If we dont document how to use it then can we really expect anyone to use it? I think we should either document how people can use it or accept that its essentially no longer available. Or convince someone to fix it of course :D

Comment (thc202) [reply site/content/faq/how-can-you-use-zap-to-scan-apis.md:16]: The blog post linked later shows how to use it, as long as you get the add-on (core help documents from where they are loaded).

**Comment by thc202** [comment]
Closing, no longer relevant the SOAP add-on is available in the marketplace (and also bundled by default).

**Pull request closed without merging**